gradle.properties

```diff
@@ -0,0 +1,27 @@
+pluginGroup = com.finanteq.plugins.idea.cucumber-kotlin-idea-plugin
+pluginName = Cucumber for Kotlin and Android Intellij Plugin
+pluginRepositoryUrl=https://github.com/Finanteq/cucumber-kotlin-intellij-plugin
+pluginVersion=242.1
+# Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
+pluginSinceBuild=242
+pluginUntilBuild=242.*
+# IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
+platformType = IC
+platformVersion=2024.2.1
+
+# Gherkin plugin version: https://plugins.jetbrains.com/plugin/9164-gherkin/versions
+platformPlugins=gherkin:242.20224.159, cucumber-java:242.20224.159, org.jetbrains.android:242.23339.11
+# Example: platformBundledPlugins = com.intellij.java
+platformBundledPlugins=com.intellij.java, org.jetbrains.kotlin
+
+# Opt-out flag for bundling Kotlin standard library -> https://jb.gg/intellij-platform-kotlin-stdlib
+kotlin.stdlib.default.dependency = false
+
+# Enable Gradle Configuration Cache -> https://docs.gradle.org/current/userguide/configuration_cache.html
+org.gradle.configuration-cache = true
```

Comment:
I don't think that these gradle specific options should be kept in repository - rather in local user `gradle.properties`